app/components/TrackComponent/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * TrackComponent
+ *
+ */
+
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import If from '@components/If';
+import { Card, Image, Typography, Button } from 'antd';
+import styled from 'styled-components';
```

Comment:
imports from dependencies should go to the top and then your component or other imports must come up

Reply:
Updated the changes accordingly